src/App/App.js

```diff
@@ -40,6 +41,7 @@ class App extends Component {
             <Route exact path="/login" component={!isAuth ? Login : Home} />
             <Route exact path="/redirect" component={Redirection} />
             <Route exact path="/signup" component={isAuth ? Home : RegistrationComponent} />
+            <Route exact path="/rate" component={Ratings} />
```

Comment:
for a user to rate a particular article he has to go to a certain route @nkpremices I don't understand why? please tell me if I am wrong

Reply:
I created this route just for design purposes @MCFrank16. Later I'm gonna bind the component to the read article component